tests/python/topi/python/test_topi_transform.py

```diff
@@ -870,6 +871,31 @@ def test_transpose():
     verify_transpose((3, 10), None)
 
 
+@tvm.testing.parametrize_targets
+def test_transpose_schedule(target, dev):
+    shape = (100, target.thread_warp_size + 3)
+    x = relay.var("x", relay.TensorType(shape, "float32"))
+    f = relay.transpose(x)
+    ex = relay.create_executor(
+        kind="graph", mod=tvm.IRModule.from_expr(relay.Function([x], f)), device=dev, target=target
+    )
+    r = np.random.rand(*shape)
+    tvm.testing.assert_allclose(ex.evaluate()(r).asnumpy(), np.transpose(r))
+
+    # We want to make sure schedule does not fire here, but there is no way of
+    # inspecting which schedules were used.
```

Comment:
Like this comment mentions, there is no way of inspecting which schedules were used, so it seems to me that the difference between this test and `test_transpose` is the workload in this test includes `add` to test the case of fusion. Accordingly, could we just extend `test_transpose`?

Reply:
We could. I like to keep it separate so the intention is known.

Reply:
Fair enough. Then it might be better to name it `test_transpose_fuse` or something like that (nit).

Reply:
switched to `test_transpose_unfused_schedule`